src/coreservices.cpp

```diff
@@ -528,11 +552,11 @@ void CoreServices::initializeKeyboard() {
 void CoreServices::slotOptionsKeyboard(bool toggle) {
     UserSettingsPointer pConfig = m_pSettingsManager->settings();
     if (toggle) {
-        //qDebug() << "Enable keyboard shortcuts/mappings";
+        // qDebug() << "Enable keyboard shortcuts/mappings";
```

Comment:
Don't reformat unrelated code. This makes your PR unnecessary big.

Reply:
I think this is precommit, and I suggest we suppress this extra space in case for often commented function like qDebug, qWarning etc.

Reply:
Pre-commit works only on changed lines. It is probably the editor that does it. It needs to be configured not to reformat the whole file on safe.

Reply:
I think it's the code editor's re-formatting coming to play here.